commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/internal/upgrade/v1_1_0/CommerceInventoryWarehouseItemUpgradeProcess.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.inventory.internal.upgrade.v1_1_0;
+
+import com.liferay.commerce.inventory.model.impl.CommerceInventoryWarehouseItemModelImpl;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Luca Pellizzon
+ */
+public class CommerceInventoryWarehouseItemUpgradeProcess
+	extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		_addColumn(
+			CommerceInventoryWarehouseItemModelImpl.class,
+			CommerceInventoryWarehouseItemModelImpl.TABLE_NAME,
+			"externalReferenceCode", "STRING");
```

Comment:
Shouldn't it be `"VARCHAR(75)"` here instead of `"STRING"`? `tables.sql` calls it a `VARCHAR(75)`.

Reply:
Copied by
com.liferay.commerce.internal.upgrade.v1_1_0.CommerceOrderNoteUpgradeProcess
So I think it needs to be fixed everywhere 😄